lettuce/flows/channel.py

```diff
@@ -0,0 +1,53 @@
+"""
+Cannel flows in 2D.
+"""
+
+import numpy as np
+import torch
+
+from lettuce.unit import UnitConversion
+
+
+class ChannelFlow2D(object):
+    def __init__(self, resolution, reynolds_number, lattice):
+        self.resolution = resolution
+        self.units = UnitConversion(
+            lattice,
+            reynolds_number=reynolds_number,
+            characteristic_length_lu=resolution, characteristic_length_pu=2*np.pi,
+            characteristic_velocity_pu=1
+        )
+
+    def analytic_solution(self, x, t=0):
+        u = np.array([x[0]*0.0, x[1]*0.0])
+        p = np.array([x[0]*0.0+0.33])
+        #p[0,:,0:70] = 0.35
+        return p, u
+
+    def initial_solution(self, x):
```

Comment:
OK, great. But this means that the flow will always start at the steady-state solution.
Maybe add a parameter `start_from_zero` to the constructor. It can be `True` by default but gives you a possibility to start from a zero velocity field.